pkg/deploy/che_configmap.go

```diff
@@ -65,6 +65,11 @@ type CheConfigMap struct {
 	PluginRegistryUrl                    string `json:"CHE_WORKSPACE_PLUGIN__REGISTRY__URL,omitempty"`
 	DevfileRegistryUrl                   string `json:"CHE_WORKSPACE_DEVFILE__REGISTRY__URL,omitempty"`
 	WebSocketEndpointMinor               string `json:"CHE_WEBSOCKET_ENDPOINT__MINOR"`
+	CheWorkspacePluginBrokerInitImage    string `json:"CHE_WORKSPACE_PLUGIN__BROKER_INIT_IMAGE,omitempty"`
+	CheWorkspacePluginBrokerUnifiedImage string `json:"CHE_WORKSPACE_PLUGIN__BROKER_UNIFIED_IMAGE,omitempty"`
+	CheServerSecureExposerJwtProxyImage  string `json:"CHE_SERVER_SECURE__EXPOSER_JWTPROXY_IMAGE,omitempty"`
+	CheWorkspaceSidecarImagePullPolicy   string `json:"CHE_WORKSPACE_SIDECAR_IMAGE__PULL__POLICY,omitempty"`
```

Comment:
> CheWorkspaceSidecarImagePullPolicy
> CheDockerAlwaysPullImage

@l0rd @nickboldt Do you know if Airgap should override these properties to force a given value ?

Reply:
yes, otherwise it'll try to pull from dockerhub/quay/rhoi repos.